src/Microsoft.AspNetCore.Mvc.RazorPages/Internal/DefaultPageApplicationModelProvider.cs

```diff
@@ -262,17 +268,25 @@ protected virtual PagePropertyModel CreatePropertyModel(PropertyInfo property)
 
             var propertyAttributes = property.GetCustomAttributes(inherit: true);
 
+            // BindingInfo for properties can be either specified by decorating the property with binding spcecific attributes
+            // of if there's binder model metadata associated with the property.
```

Comment:
Other than spelling "specific" correctly (congratulations!), suggest wording this comment as I suggested in `DefaultApplicationModelProvider`.